docs/json/radarr/cf/hdr10.json

```diff
@@ -10,7 +11,7 @@
       "negate": false,
       "required": true,
       "fields": {
-        "value": "\\bHDR10(\\b[^+|Plus])"
+        "value": "((\\bHDR10)|(\\b10bit\\WHDR))(\\b[^+|Plus])"
```

Comment:
10bit doesn't always mean it's HDR10
The scene uses HDR in their release name where it can mean the following:
- [HDR10](https://trash-guides.info/Radarr/Radarr-collection-of-custom-formats/#hdr10)
- [PQ](https://trash-guides.info/Radarr/Radarr-collection-of-custom-formats/#pq)
- And in worse cases if the release group screwed up [HDR10+](https://trash-guides.info/Radarr/Radarr-collection-of-custom-formats/#hdr10plus)+

The release you have as example would be picked up as [HDR (undefined)](https://trash-guides.info/Radarr/Radarr-collection-of-custom-formats/#hdr-undefined)

After the Starr apps analyzed the file it will use the correct HDR format for the imported and renamed file.

Reply:
Thank you for your information